### ISSUE TITLE
[26-12-18 Snapshot] Just a few animation bugs

### ISSUE THREAD

**Issue by JordanCJbrowne** [iwoplaza/MoBends#53]
The big one, players dont paddle nor do anything with their arms while riding in a minecart.

![2019-03-12_15 54 19](https://user-images.githubusercontent.com/47202371/54233777-10681680-44e4-11e9-8876-5ce6f9e7e8c8.png)
![2019-03-12_15 12 01](https://user-images.githubusercontent.com/47202371/54233796-18c05180-44e4-11e9-84d0-77d18ea72fd5.png)
![2019-03-12_02 15 43](https://user-images.githubusercontent.com/47202371/54233810-1eb63280-44e4-11e9-85cc-56fc05077101.png)

For this one I HIGHLY recommend an animation where the player has their arms bent at the elbow, and the elbow resting on the side of the boat/minecart. The players hands would then line up with the paddles and you could do an animation where the hands push foward, the elbow would glide along the edge of the boat with the hands, and then they would glide back smoothly.

Another slightly minor one, players just stand still while on fire. Itd be really cool to see an animation of the character flailing their arms in the air or trying to brush themselves off super fast!
![2019-03-12_15 35 47](https://user-images.githubusercontent.com/47202371/54233948-663cbe80-44e4-11e9-8d84-7b0f073d284c.png)

And then my final recommendation is the animation for opening clickable inventories (However I did realize the player will lift/drop the chest lid). When clicking on a crafting table/anvil/furnace the player sort of takes a stance like he is going to fight, which I'm guessing because the mod does think the player is trying to swing fists. It would be cool to see the player slightly bend over and place their hands on the table/anvil/furnace/any clickable inventory. Like if you've ever seen Skyrim?  The way they place their hands on the enchantment table would be perfect in this mod.
![2019-03-12_15 25 51](https://user-images.githubusercontent.com/47202371/54234178-f24ee600-44e4-11e9-9af6-bea0b1a99d88.png)

{You cant really see but there is an anvil in front of me I just clicked on}

An animation like this (below) when accessing those types of blocks would show a lot more focus coming from the character rather than thinking they are going to beat the living shit out of their crafting table xD
![1114157312](https://user-images.githubusercontent.com/47202371/54234463-8de05680-44e5-11e9-8066-55b859b9a3de.jpg)

**Comment by JordanCJbrowne** [comment]
Yeah right clicking almost anything leads to that "Prepared to fight" animation for the first few seconds, such as sitting.

**Comment by iwoplaza** [comment]
Thanks for pointing these out 😄. I'll definetely come back to this and try to implement as many of these ideas as I can.
